azure-pipelines.yml

```diff
@@ -516,6 +520,22 @@ jobs:
     enabled: True
     inputs:
       tags: 'BuildConfiguration: ${{ parameters.BuildConfiguration }}'
+
+  - task: SdtReport@2
+    inputs:
+      GdnExportAllTools: true
```

Comment:
Suggestion/idea here is:
```inputs:
       GdnExportAllTools: false
       GdnExportHtmlFile: true
       GdnExportOutputBaselineFile: myBaseline
       GdnExportOutputBaseline: myBaselinedResults
       GdnExportOutputSuppressionFile: mySuppressions
       GdnExportOutputSuppressionSet: mySuppressionSet
       GdnExportGdnToolCodesignValidationSeverity: Warning
```

baseline file format/schema can be found [here](https://eng.ms/docs/microsoft-security/security/azure-security/cloudai-security-fundamentals-engineering/security-integration/guardian-wiki/microsoft-guardian/general/resource-samples/baseline-file), it is easily readable.

Reply:
So i had decided to keep it simple because our use case if pretty simple and want to avoid complexity. For example we dont have any need for suppressions, so kept that out. Code sign validation severity should be error hence didnt update default of that setting. What do you see the baseline being used for?

Reply:
baseline is just a json which is human readable to see which tool failed and why it failed.

Reply:
Tried some of these flags, but postanalysis still didnt include error logs in that task itself.